WinUIGallery/Samples/ControlPages/CaptureElementPreviewPage.xaml.cs

```diff
@@ -55,30 +60,68 @@ private void CaptureElementPreviewPage_Unloaded(object sender, RoutedEventArgs e
     private MediaFrameSourceGroup mediaFrameSourceGroup;
     private MediaCapture mediaCapture;
 
-    async private void StartCaptureElement()
+    private async Task StartCaptureElement()
```

Comment:
I think it would be better to have this method really just start the capturing and have the caller of this method handle errors.

Reply:
You are right. It's better this way.